assets/css/style.css

```
@@ -22,4 +22,29 @@ body:not(.is-web) .product-hunt-badge {display:none;}
+    --es-option-min-width:250px;
+    --es-scrollbar-width:3px;
+    --es-scrollbar-track:var(--color-secondary);
+    --es-scrollbar-thumb:var(--color-secondary-hightlight);
+    --es-bg:var(--color-secondary);
+    --es-bg-hover:var(--color-primary);
+    --es-text:var(--color-on-secondary);
+    --es-text-hover:var(--color-on-primary);
+    --es-border:1px solid var(--color-secondary-hightlight);
+    --es-border-hover:1px solid var(--color-primary);
+    --es-radius:5px;
+    transition:var(--transition);
```

Stylelint (17.2.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)